src/layout/_template.njk

```diff
@@ -243,4 +248,4 @@
 
         {% block scripts %}{% endblock %}
     </body>
-</html>
+</html>
```

Comment:
This line at the end shouldnt have been removed

Reply:
For some reason git is showing the line being removed and then added back in the same place. So looks like the line is still there